### PR TITLE
Test 1

### PR DESCRIPTION
**Pull request by Naryngul**
My 1st test

### CODE REVIEW

**Comment by Naryngul** [comment]
Test Pull

**Comment by MrAzat** [comment]
Great Naryngul! Very useful information

File: admin/admin-knowledge-checklist.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ## Admin's Knowledge Checklist
 
-Checklist of 10 concepts every Salesforce admin must know:
+Checklist of 10 concepts every Salesforce admin must know
+First Test
+2nd test
+
```
Comment (usmanovs) [inline admin/admin-knowledge-checklist.md:6]: why add a blank line?

File: admin/admin-knowledge-checklist.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ## Admin's Knowledge Checklist
 
-Checklist of 10 concepts every Salesforce admin must know:
+Checklist of 10 concepts every Salesforce admin must know
+First Test
```
Comment (usmanovs) [inline admin/admin-knowledge-checklist.md:4]: brilliant suggestion!

**Review by usmanovs — COMMENTED** [review]
nice insight.

**Pull request closed without merging**